package.json

```diff
@@ -21,8 +21,8 @@
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev --parallel",
-    "ci:test-coverage": "NODE_ENV=test run-s services:run-test contracts test:coverage services:clean-test",
-    "ci:test": "NODE_ENV=test run-s services:run-test contracts test services:clean-test",
+    "ci:test-coverage": "NODE_ENV=test CI=true run-s services:run-test contracts test:coverage services:clean-test",
```

Comment:
you can add this in the action instead of here

Reply:
we need to be able to run it locally too, better to keep in scripts